jumpcloud-ADMU/Powershell/Private/JumpCloudFunctions/Get-MtpOrganization.ps1

```diff
@@ -29,12 +29,20 @@ function Get-MtpOrganization {
             try {
                 Set-JcUrl -Region "US"
                 $baseUrl = "$($global:JCUrl)/api/organizations/$($orgID)"
-                $Request = Invoke-WebRequest -Uri "$($baseUrl)?limit=$($limit)&skip=$($skip)" -Method Get -Headers $Headers -UseBasicParsing
+                try {
```

Comment:
I do want to clean this up in a future card, there are a few functions doing similar work but for this card, I wanted this function to throw when there was an invalid API key passed in. This allows for more comprehensive testing.